ruby_on_rails/rspec.md

```diff
@@ -122,7 +122,9 @@ module JavaScriptErrorCollector
     aggregate_failures 'javascript errors' do
       errors.each do |error|
         expect(error.level).not_to eq('SEVERE'), error.message
-        expect(error.level).not_to eq('WARNING'), error.message
+
+        next unless error.level == 'WARNING'
+        puts "\e[33m\nJAVASCRIPT WARNING\n#{error.message}\e[0m"
```

Comment:
I'd not show those in the console. This would pollute our nice green dots.

I'd leave this away except someone opts-in with a config setting.